corehq/apps/cachehq/cachemodels.py

```diff
@@ -76,7 +76,6 @@ class LocationGenerationCache(GenerationCache):
     generation_key = '#gen#location#'
     doc_types = ['Location']
     views = [
-        'commtrack/locations_by_code',
         '_all_docs',
```

Comment:
`_all_docs` is a view-like endpoint built into couchdb that is used by e.g. `get_docs` (that maybe wasn't your question.)

It looks like this was added because of the `Location.view('_all_docs', ...` which you've deleted below, so I'd say this is fair game to remove as well.